pages/care-package/brokerage/package-details/[id].js

```diff
@@ -0,0 +1,175 @@
+import React, { useState } from 'react';
```

Comment:
This page was moved to new url - `/care-package/[guid]/review`. Please make sure that it will stay on this url. Also note that `query.id` was changed to `query.guid`